src/phoenix/experiments/types.py

```diff
@@ -219,14 +219,14 @@ class ExperimentRun:
     experiment_id: ExperimentId
     dataset_example_id: ExampleId
     repetition_number: RepetitionNumber
-    output: Optional[ExperimentResult] = None
+    experiment_run_output: ExperimentRunOutput
     error: Optional[str] = None
     id: ExperimentRunId = field(default_factory=_dry_run_id)
     trace_id: Optional[TraceId] = None
 
     @property
-    def task_output(self) -> Optional[TaskOutput]:
-        return deepcopy(self.output.result) if self.output else None
+    def output(self) -> Optional[TaskOutput]:
+        return deepcopy(self.experiment_run_output.task_output)
```

Comment:
Do we need the entire `experiment_run_output` on `ExperimentRun` if we only use the `task_output`? Seems like we could just have `output` as the `task_output`.

Reply:
See my other comment